include/Reader/reader.h

```diff
@@ -3241,6 +3251,13 @@ class ReaderBase {
   virtual bool arrayGet(CORINFO_SIG_INFO *Sig, IRNode **RetVal) = 0;
   virtual bool arraySet(CORINFO_SIG_INFO *Sig) = 0;
 
+  /// \brief Check whether structs are represented by pointers on the operand
+  /// stack.
+  ///
+  /// \returns true iff structs are represented by pointers on the operand
+  /// stack.
+  virtual bool structsAreRepresentedByPointers() = 0;
```

Comment:
I suppose it's ok, my guess is the factoring between reader base and reader is off somewhere and reader base should just be told this and not have to know it (via something like the `IsValueIsPointer` parameter), but let's not worry about it now.